examples/container-service-extension/3.7-cse-step1.tf

```diff
@@ -0,0 +1,274 @@
+# If you have already an Organization, you can remove the `vcd_org` resource from this HCL file and simply
+# configure it in the provider settings.
+
```

Comment:
also maybe worth to mentioned that Tenant role is not enough

Reply:
Done